app/src/main/java/com/bunbeauty/papakarlo/feature/menu/state/MenuViewState.kt

```diff
@@ -0,0 +1,45 @@
+package com.bunbeauty.papakarlo.feature.menu.state
+
+import androidx.compose.runtime.Immutable
+import com.bunbeauty.papakarlo.feature.topcart.TopCartUi
+import com.bunbeauty.shared.presentation.menu.model.CategoryItem
+import com.bunbeauty.shared.presentation.menu.model.MenuDataState
+import kotlinx.collections.immutable.ImmutableList
+
+// TODO refactor
```

Comment:
comment?

Reply:
MenuViewState needs refactoring
It will be done in a separate PR.